### PR TITLE
SW-4522 Add project name to the report view and edit

### PR DESCRIPTION
**Pull request by nickgraz**
- Add project name to the report view and edit

- Display nursery, planting site, and seed bank name (instead of checkbox) when it is a project report

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #2015** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2015?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2015?utm_source=stack-comment).

File: src/components/Reports/ReportForm.tsx [hunk 1]
```diff
@@ -233,14 +237,18 @@ export default function ReportForm(props: ReportFormProps): JSX.Element {
...
+                ) : (
+                  <Checkbox
+                    id={`seedbank-${index}`}
+                    disabled={!editable}
+                    name={seedbank.name}
+                    label={seedbank.name}
+                    value={seedbank.selected}
+                    onChange={(value) => handleAddRemoveLocation(value, index, 'seedBanks')}
+                  />
+                )}
```
Comment (karthikbtf) [inline src/components/Reports/ReportForm.tsx:251]: extract out for reusability?

**Review by karthikbtf — APPROVED** [review]
one suggestion

File: src/components/Reports/ReportForm.tsx [hunk 2]
```diff
@@ -102,6 +102,32 @@ export default function ReportForm(props: ReportFormProps): JSX.Element {
   const smallItemGridWidth = () => (isMobile ? 12 : 4);
   const mediumItemGridWidth = () => (isMobile || isTablet ? 12 : 8);
 
+  const LocationCheckbox = useCallback(
```
Comment (karthikbtf) [inline src/components/Reports/ReportForm.tsx:104]: awesome, thanks!

**Review by karthikbtf — APPROVED** [review]

**Comment by nickgraz** [comment]
### Merge activity

* **Jan 12, 3:39 PM**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2015).

**Pull request merged**